packages/runtime/datastore/src/localChannelContext.ts

```diff
@@ -92,8 +94,8 @@ export class LocalChannelContext implements IChannelContext {
     }
 
     public setConnectionState(connected: boolean, clientId?: string) {
-        // Connection events are ignored if the data store is not yet attached
-        if (!this.attached) {
+        // Connection events are ignored if the data store is not yet attached or loaded
+        if (!(this.attached && this.isLoaded)) {
```

Comment:
I think code is more readable if it's in the form of 
```
if (this.attached && this.isLoaded) {
   this.services.value.deltaConnection.setConnectionState(connected);
}
```
early returns are great if there a lot of nested IFs or code block is long, but for one-liner - not so much

Reply:
Changed